tests/test-sessions/test-hazelcast-sessions/pom.xml

```diff
@@ -76,16 +83,10 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-hazelcast</artifactId>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>com.hazelcast</groupId>
-          <artifactId>hazelcast</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
     <dependency>
        <groupId>com.hazelcast</groupId>
-       <artifactId>hazelcast-all</artifactId>
+       <artifactId>hazelcast</artifactId>
```

Comment:
Heh. Well, the whole module is test module, so `scope` test is kinda funny anyway.

Reply:
don't know maybe it's wrong but all test-sessions are using such pattern.